src/background.ts

```diff
@@ -125,55 +125,66 @@ const electronStoreJsonSchema = zodToJsonSchema(electronStoreSchema);
-      const prevDefaultStyleIds = store.get("defaultStyleIds");
-      store.set(
-        "defaultStyleIds",
-        prevDefaultStyleIds.map((defaultStyle) => ({
-          engineId,
-          speakerUuid: defaultStyle.speakerUuid,
-          defaultStyleId: defaultStyle.defaultStyleId,
-        }))
-      );
+let store: Store<ElectronStoreType>;
+try {
+  store = new Store<ElectronStoreType>({
+    schema: electronStoreJsonSchema.properties as Schema<ElectronStoreType>,
+    migrations: {
+      ">=0.13": (store) => {
+        // acceptTems -> acceptTerms
+        const prevIdentifier = "acceptTems";
+        const prevValue = store.get(prevIdentifier, undefined) as
+          | AcceptTermsStatus
+          | undefined;
+        if (prevValue) {
+          // eslint-disable-next-line @typescript-eslint/no-explicit-any
+          store.delete(prevIdentifier as any);
+          store.set("acceptTerms", prevValue);
+        }
+      },
+      ">=0.14": (store) => {
+        // FIXME: できるならEngineManagerからEnginIDを取得したい
+        if (process.env.DEFAULT_ENGINE_INFOS == undefined)
+          throw new Error("DEFAULT_ENGINE_INFOS == undefined");
+        const engineId = EngineId(
+          JSON.parse(process.env.DEFAULT_ENGINE_INFOS)[0].uuid
+        );
+        if (engineId == undefined)
+          throw new Error("DEFAULT_ENGINE_INFOS[0].uuid == undefined");
+        const prevDefaultStyleIds = store.get("defaultStyleIds");
+        store.set(
+          "defaultStyleIds",
+          prevDefaultStyleIds.map((defaultStyle) => ({
+            engineId,
+            speakerUuid: defaultStyle.speakerUuid,
+            defaultStyleId: defaultStyle.defaultStyleId,
+          }))
+        );
 
-      const outputSamplingRate: number =
+        const outputSamplingRate: number =
+          // @ts-expect-error 削除されたパラメータ。
+          store.get("savingSetting").outputSamplingRate;
+        store.set(`engineSettings.${engineId}`, {
+          useGpu: store.get("useGpu"),
+          outputSamplingRate:
+            outputSamplingRate === 24000 ? "engineDefault" : outputSamplingRate,
+        });
         // @ts-expect-error 削除されたパラメータ。
-        store.get("savingSetting").outputSamplingRate;
-      store.set(`engineSettings.${engineId}`, {
-        useGpu: store.get("useGpu"),
-        outputSamplingRate:
-          outputSamplingRate === 24000 ? "engineDefault" : outputSamplingRate,
-      });
-      // @ts-expect-error 削除されたパラメータ。
-      store.delete("savingSetting.outputSamplingRate");
-      // @ts-expect-error 削除されたパラメータ。
-      store.delete("useGpu");
+        store.delete("savingSetting.outputSamplingRate");
+        // @ts-expect-error 削除されたパラメータ。
+        store.delete("useGpu");
+      },
     },
-  },
-});
+  });
+} catch (e) {
+  dialog.showErrorBox(
+    "設定ファイルの読み込みに失敗しました。",
+    `${app.getPath(
+      "userData"
+    )} にある config.json の名前を変えることで解決することがあります（ただし設定がすべてリセットされます）。`
+  );
+  throw e;
```

Comment:
```suggestion
  app.exit();
  throw e;
```

バックグラウンドで残っちゃうのでapp.exitが必要かと。（throwはTypeScript用、ないと下の方で型エラーを吐く）

Reply:
throw だけでも、typecheck 自体は通ってましたが、app.exit もすることにします。